# HG changeset patch
# User Rik <rik@octave.org>
# Date 1546723715 28800
#      Sat Jan 05 13:28:35 2019 -0800
# Branch stable
# Node ID 88dd00f48522ed8b28aa5c605868c4b70b958d2c
# Parent  ac6f0b8e94c0fcae860ed8203dc1a057725b5601
Fix compilation error in 73ce195bf66e.

* ls-oct-text.cc (save_three_d): Ask compiler to explicitly convert stream to
bool value.

diff --git a/libinterp/corefcn/ls-oct-text.cc b/libinterp/corefcn/ls-oct-text.cc
--- a/libinterp/corefcn/ls-oct-text.cc
+++ b/libinterp/corefcn/ls-oct-text.cc
@@ -393,17 +393,17 @@ save_three_d (std::ostream& os, const oc
           os << tmp.extract (0, i, nr-1, i);
           if (i+1 < nc)
             os << "\n";
         }
     }
 
   os.precision (old_precision);
 
-  return (os);
+  return (static_cast<bool> (os));
 }
 
 DEFUN (save_precision, args, nargout,
        doc: /* -*- texinfo -*-
 @deftypefn  {} {@var{val} =} save_precision ()
 @deftypefnx {} {@var{old_val} =} save_precision (@var{new_val})
 @deftypefnx {} {} save_precision (@var{new_val}, "local")
 Query or set the internal variable that specifies the number of digits to
